autoload/wiki/buffer.vim

```diff
@@ -128,7 +128,7 @@ function! s:init_buffer_mappings() abort " {{{1
 
 
   let l:mappings = {}
-  if get(g:, 'wiki_mappings_use_defaults', 1)
+  if get(g:, 'wiki_mappings_use_all_defaults', 1) || get(g:, 'wiki_mappings_use_local_defaults', 0)
```

Comment:
`get()` uses 0 as default value, so no need to use `, 0)` here.

Reply:
Also, the line is too long. I want to keep things at 80 cols. So, change it to:

```vim
   if get(g:, 'wiki_mappings_use_all_defaults', 1)
        \ || get(g:, 'wiki_mappings_use_local_defaults')
```

Reply:
Fixed in a06e342